contribute/process/pr-workflow-developer-doc.md

```diff
@@ -0,0 +1,40 @@
+# Contributing a Patch to Developer Documentation
+
+Developer documentation is available under the openebs/openebs/contribute folder. 
+
+The developer documenation includes:
+- Architecture and Design documentation
+- Technical or Research Notes
+- FAQ
+- Process Documentation 
+- Generic / Miscellaneous Notes 
+
+Anything that will help other developers!
+
+## Submit an Issue describing your proposed change.
```

Comment:
reference here to creating an issue where labels for doc issues can be labelled  "documentation"